tests/test_io.py

```diff
@@ -149,10 +149,10 @@ def test_from_qasm_deprecated(self, mock_plugin_converters):
         """Tests that the current default behaviour of from_qasm is deprecated."""
 
         with pytest.warns(qml.PennyLaneDeprecationWarning, match="The current default behaviour"):
-            qml.from_qasm("Test")
+            qml.from_qasm("measure q[i] -> c[i];")
 
         called_args, called_kwargs = mock_plugin_converters["qasm"].call_args
-        assert called_args == ("Test",)
+        assert called_args == ("measure q[i] -> c[i];",)
         assert called_kwargs == {"measurements": []}
```

Comment:
I suppose the key purpose of this PR is to _not_ raise a warning when no measurements are present, right? So might be good to add a test specifically for that scenario, asserting that no warning is raised.

Reply:
The `PennyLaneDeprecationWarning` is treated as an error and would fail the test case if it were raised. There are other test cases that calls `from_qasm` with circuits containing no measurements, so that essentially asserts that no warning is raised in those cases.

Reply:
Ah, I was missing this bit, thank you!